assets/js/components/settings/SettingsCardKeyMetrics.js

```diff
@@ -61,6 +65,12 @@ export default function SettingsCardKeyMetrics() {
 	const hasUserPickedMetrics = useSelect( ( select ) =>
 		select( CORE_USER ).getUserPickedMetrics()
 	);
+	const isACRNotificationDismissed = useSelect( ( select ) =>
+		select( CORE_USER ).isItemDismissed( ACR_SUBTLE_NOTIFICATION_SLUG )
+	);
+	const showACRNotification =
+		conversionReportingEnabled && ! isACRNotificationDismissed;
```

Comment:
If `isACRNotificationDismissed` is `undefined` because the value is still loading, this might set `showACRNotification` as `true` but then once it's loaded and `isACRNotificationDismissed` is set to `true`, it will change to `false`. That would trigger a brief "flash" of the notification component before it's hidden, so let's update this to:

```suggestion
	const showACRNotification =
		conversionReportingEnabled && isACRNotificationDismissed === false;
```

Maybe we want a test for this behaviour as well 😄